### DISCUSSION TITLE
What is the small colored label in the README image?

### DISCUSSION THREAD

**Discussion by rick-200** [Eugeny/tabby#7837] [Q&A] [answered]
Hello,
I really want to know what is the small colored label in the README image, and whether I can turn this feature on.

<img width="1240" alt="readme-terminal" src="https://user-images.githubusercontent.com/25481708/214837348-514f2a18-9d9d-4957-b7a7-b243e6b52424.png">

**Comment by Eugeny** [comment, marked as answer]
It's the default theme from https://ohmyz.sh/

**Reply by rick-200** [reply]
Thanks!

**Comment by lawweiliang** [comment]
@rick-200 
Please close the discussion if the question was answered. 
It would be helpful to make the discussion section clean. 

Click `mark as answered` and `close the discussion`. 😄 

![image](https://user-images.githubusercontent.com/34936929/228098333-222ac3d4-4c39-4c26-a2a8-52906647e00e.png)
